tests/test_utils.py

```
@@ -0,0 +1,70 @@
+"""Tests for utility functions in the power consumption project."""
+
+from pathlib import Path
```

Ruff

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)